packages/react-refresh/src/ReactFreshBabelPlugin.js

```diff
@@ -11,7 +11,7 @@ export default function(babel) {
   if (typeof babel.getEnv === 'function') {
     // Only available in Babel 7.
     const env = babel.getEnv();
-    if (env !== 'development') {
+    if (env !== 'development' && typeof expect !== 'function') {
```

Comment:
@gaearon My understanding was that the purpose of this error is to prevent people from enabling the Fresh plugin in their unit tests. Doesn't this change mean that consumers of this plugin who use Jest won't get the error below?

Reply:
We actually do run it in tests in fbsource. So I don't know that it's necessarily a bad idea.

The primary purpose is to prevent people from including it in prod setups.